src/FluxCalculator.jl

```diff
@@ -6,15 +6,24 @@ or to call flux calculating functions from the component models.
 """
 
 module FluxCalculator
+import SurfaceFluxes as SF
+import Thermodynamics as TD
+using StaticArrays
 
-using ClimaCoupler.Interfacer
-using ClimaCore.Fields
+using ClimaCoupler: Interfacer, Regridder
+using ClimaCore: Fields, Spaces
 export PartitionedComponentModelGrid,
     CombinedAtmosGrid,
-    compute_combined_turbulent_fluxes!,
+    combined_turbulent_fluxes!,
```

Comment:
If we're renaming this function, should we also rename `compute_atmos_turbulent_fluxes!` to `atmos_turbulent_fluxes!`?

Reply:
Sure, I think it's clear from the function signature anyway :)